edi_product_oca/__manifest__.py

```diff
@@ -4,14 +4,20 @@
 {
     "name": "EDI Product",
     "summary": """
-       EDI framework configuration and base logic for products""",
+       EDI framework configuration and base logic for products, product packagings
+       and product barcodes.
+       """,
     "version": "16.0.1.0.0",
     "license": "AGPL-3",
     "author": "ForgeFlow, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/edi-framework",
     "depends": [
         "product",
+        "product_multi_barcode",
```

Comment:
this should go to its own module (eg: edi_product_multi_barcode) w/ auto_install=True

Reply:
Thank @simahawk , I have created a PR for new [edi_product_multi_barcode_oca](https://github.com/OCA/edi-framework/pull/54)